src/pymovements/events/events.py

```diff
@@ -423,7 +425,21 @@ def copy(self) -> Events:
         """
         return self.clone()
 
-    def split(self, by: Sequence[str] | None = None) -> list[Events]:
+    @overload
+    def split(self, by: Sequence[str] | None = None, *, as_dict: Literal[False]) -> list[Events]:
+        ...
+
```

Comment:
the newlines between the overloaded signatures are automatically added by `autopep8`

Reply:
because of line length